### DISCUSSION TITLE
Fuseki dataset edit page

### DISCUSSION THREAD

**Discussion by royassis** [apache/jena#2312] [Q&A]
Hey

Can someone explain or point me to information about the dataset edit page?

![image](https://github.com/apache/jena/assets/34189329/5ec5dd32-0f1f-4ede-b039-da682b2b4233)

**Comment by kinow** [comment]
I think we do not have a documentation page about that page. What's your question?

I rarely use this view, but it is used to list current graphs, and let you preview or modify the graph. It does not work for large graphs - cannot remember what's the threshold, but the UI warns you. You must also have a service for writing the graph, as otherwise the UI fails complaining it can only read your data.

**Comment by royassis** [comment]
Hey @kinow. I would like to know what kind of edits I can do using this page. Maybe you can also add and example, and an explanation about the syntax.

**Reply by royassis** [reply]
Ok I now understand. 
The input for the box at the right should be a .ttl formatted data.
